app/models/vm_or_template/right_sizing.rb

```diff
@@ -31,8 +31,57 @@ module VmOrTemplate::RightSizing
 
     virtual_column :max_cpu_usage_rate_average_max_over_time_period,     :type => :float
     virtual_column :max_mem_usage_absolute_average_max_over_time_period, :type => :float
-    virtual_column :cpu_usagemhz_rate_average_max_over_time_period,      :type => :float
-    virtual_column :derived_memory_used_max_over_time_period,            :type => :float
+
+    virtual_attribute :cpu_usagemhz_rate_average_max_over_time_period,
+                      :float, :arel => (lambda do |t|
+                        col = :cpu_usagemhz_rate_average
+                        metric_rollup_table = MetricRollup.arel_table
+                        select_clause = metric_rollup_table[col].maximum
```

Comment:
Oh, definitely.  I skipped the "make this DRY" in this since I didn't expect this to be merged as is.  The `col` definition in each of these blocks should be the only difference between the two (though, it has been a week since I wrote this and looked at it now...)